docs/README.fr.md

```diff
@@ -0,0 +1,35 @@
+<h3 align="center"><img src="../assets/onefetch.svg" height="130px"></h3>
+
+<h5 align="center">Un outil en ligne de commande affichant les informations Git écrit en Rust</h5>
+
+<p align="center">
+	<a href="https://crates.io/crates/onefetch"><img src="https://img.shields.io/crates/v/onefetch.svg" alt="cargo"></a>
+	<a href="https://github.com/o2sh/onefetch/actions"><img src="https://github.com/o2sh/onefetch/workflows/CI/badge.svg" alt="Build Status"></a>
+  <a href="https://github.com/o2sh/onefetch/wiki/language-Support"><img src="../assets/language-badge.svg"></a>
+	<a href="https://github.com/o2sh/onefetch/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22"><img src="https://img.shields.io/github/issues/o2sh/onefetch/help%20wanted?color=green" alt="help wanted"></a>
+	<a href="../LICENSE.md"><img src="https://img.shields.io/badge/license-MIT-blue.svg"></a>
+	<img src="../assets/msrv-badge.svg">
+</p>
+
+<p align="center">
+  <a href="docs/README.ja.md">日本語</a> | <a href="docs/README.fa.md">فارسی</a> |
+  <a href="docs/README.cn.md">简体中文</a> | <a href="docs/README.ru.md">Русский</a> |
+  <a href="docs/README.es.md">Español</a> | <a href="docs/README.fr.md">Français</a>
+</p>
```

Comment:
You can remove this part: we chose to only keep the translation hyperlinks in the main `README.md`.